scheduler/scheduler-examples/src/main/java/org/ow2/proactive/scheduler/examples/FTPConnector.java

```diff
@@ -67,15 +68,25 @@
 
     private boolean ftpExtractArchive;
 
-    private String ftpLocalRelativePath = null;
+    private String ftpLocalRelativePath;
 
-    private String ftpRemoteRelativePath = null;
+    private String ftpRemoteRelativePath;
 
-    private String ftpMode = null;
+    private String ftpMode;
 
-    private String ftpUsername = null;
+    private String ftpUsername;
 
-    private String ftpPassword = null;
+    private String ftpPassword;
+
+    private String ftpUrlKey;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "ftpUrlKey" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)